tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CommencementDateControllerTests.cs

```diff
@@ -22,6 +24,14 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
     [SuppressMessage("ReSharper", "NUnit.MethodWithParametersAndTestAttribute", Justification = "False positive")]
     internal static class CommencementDateControllerTests
     {
+        private static Mock<IOrderService> moqorderService;
+
+        [SetUp]
+        public static void Setup()
+        {
+            moqorderService = new Mock<IOrderService>();
+        }
```

Comment:
We try to avoid shared state and set up methods in our tests.

The `IOrderService` mock should be injected into each test using AutoFixture.  There should be some examples of this in existing tests, for example in `CompleteOrderServiceTests`.

Reply:
Updated approach to use `[Frozen] Mock<IOrderService> orderService`